reports/views.py

```diff
@@ -74,20 +81,19 @@ def make_filter(my_request):
     return query_attrs
 
 
-class ReportHome(TemplateView):
-    """
-    List of available reports
-    """
-    template_name = 'report.html'
-
-    def get_context_data(self, **kwargs):
-        context = super(ReportHome, self).get_context_data(**kwargs)
-        form = FilterForm()
-        context['form'] = form
+class IndicatorTrackingHome(ListView):
+    '''
+        Main home page for indicator tracking table
```

Comment:
@Kimaiyo077 can you rename from `indicator tracking table` to ~`Indicator Performance Tracking Table`?